examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -52,10 +52,13 @@ async function start(): Promise<void> {
 
     // Get or create the document depending if we are running through the create new flow
     const client = useFrs ? FrsClient : TinyliciousClient;
-    const [fluidContainer, containerServices] = createNew
+    const containerAndServices = createNew
```

Comment:
Nit: optionally you could have this in one line
```suggestion
    const { fluidContainer, containerServices } = createNew
```